test/functional/README.md

```diff
@@ -0,0 +1,372 @@
+## Functional test suite
```

Comment:
The test/functional/README.md file should be linked in the root README.md file, from the right place.

Reply:
Added here: https://github.com/telefonicaid/iotagent-node-lib/pull/1525/commits/05b86a5e4bfecc4cce4f320b810334a4b590fc64